container_images.mk

```diff
@@ -25,3 +25,8 @@ KUBE_SCHEDULER_IMAGE_VERSION = 1.11.7
 NGINX_IMAGE = nginx@sha256:dd2d0ac3fff2f007d99e033b64854be0941e19a2ad51f174d9240dda20d9f534
 NGINX_IMAGE_NAME = nginx
 NGINX_IMAGE_VERSION = 1.15.8
+
+# Custom built images
+SALT_MASTER_IMAGE = ""
+SALT_MASTER_IMAGE_NAME = salt-master
+SALT_MASTER_IMAGE_VERSION = 1.0.0
```

Comment:
No. This depends on the version of `salt-master` we build. This can be passed in as an `ARG` during the image build phase.